daemon/api.go

```diff
@@ -585,21 +585,50 @@ func deleteOp(c *Command, r *http.Request) Response {
 	}
 }
 
+type licT struct {
```

Comment:
Can we name this something slightly more clear, like licenseType, or snapLicense, or licenseAgreement, or something else of your preference?